lib/standards/aria-roles.js

```diff
@@ -17,13 +18,13 @@
 */
 const ariaRoles = {
   alert: {
-    type: 'widget',
+    type: 'structure',
```

Comment:
This creates a problem for `focus-order-semantics`. With this change, the following is going to start failing. I don't think we should want that.

```html
<div role="alert" tabindex="0"> Hello world </div>
```

This is what we talked about on Thursday. We'll need some way to exempt these roles that are no longer considered as widgets in the `focus-order-semantics` rule. I think the best way to do that is just to add them to the `VALID_ROLES_FOR_SCROLLABLE_REGIONS` object in `valid-scrollable-semantics-evaluate`.